lib/reporting/identity_verification_report.rb

```diff
@@ -282,39 +323,43 @@ def data
           event = row['name']
           user_id = row['user_id']
           success = row['success']
+          gpo_verification_pending = row['gpo_verification_pending'] == '1'
+          in_person_verification_pending = row['in_person_verification_pending'] == '1'
+          fraud_review_pending = row['fraud_review_pending'] == '1'
 
-          event_users[event] << user_id
+          ignore_event_for_user =
+            fraud_review_pending &&
+            EVENTS_TO_IGNORE_IF_FRAUD_REVIEW_PENDING.include?(event)
+
+          users[event] << user_id unless ignore_event_for_user
+          users[sp_key(row['service_provider'])] << user_id if row['service_provider'].present?
 
           case event
           when Events::IDV_FINAL_RESOLUTION
-            event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id if row['identity_verified'] == '1'
-
-            gpo_verification_pending = row['gpo_verification_pending'] == '1'
-            in_person_verification_pending = row['in_person_verification_pending'] == '1'
-            fraud_review_pending = row['fraud_review_pending'] == '1'
+            users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id if row['identity_verified'] == '1'
 
             if !gpo_verification_pending && !in_person_verification_pending
-              event_users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if fraud_review_pending
+              users[Results::IDV_FINAL_RESOLUTION_FRAUD_REVIEW] << user_id if fraud_review_pending
             elsif gpo_verification_pending && !in_person_verification_pending
-              event_users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if !fraud_review_pending
-              event_users[Results::IDV_FINAL_RESOLUTION_GPO_FRAUD_REVIEW] << user_id if fraud_review_pending
+              users[Results::IDV_FINAL_RESOLUTION_GPO] << user_id if !fraud_review_pending
+              users[Results::IDV_FINAL_RESOLUTION_GPO_FRAUD_REVIEW] << user_id if fraud_review_pending
```

Comment:
Note that previously we were not correctly calculating the `Workflow completed - GPO Pending - Fraud Review` metric, since we were not previously pulling GPO events marked as pending fraud review.

The changes on this branch move a small portion of the `Workflow completed - GPO Pending` stat into `Workflow completed - GPO Pending - Fraud Review`, but overall number remain the same.

Reply:
A similar issue exists with the `Workflow completed - In-Person Pending - Fraud Review` metric--we weren't previously pulling those records, so this number was always 0.